core/drivers/crypto/se050/glue/include/i2c_native.h

```diff
@@ -13,4 +13,17 @@ TEE_Result native_i2c_transfer(struct rpc_i2c_request *req,
 			       size_t *bytes);
 int native_i2c_init(void);
 
+#if CFG_CORE_SE05X_I2C_TRAMPOLINE
+TEE_Result native_i2c_transfer(struct rpc_i2c_request *req,
+			       size_t *bytes)
+{
+	/* Stub only */
+	return TEE_SUCCESS;
+}
+
+int native_i2c_init(void)
+{
+	/* Stub only */
+	return 0;
+}
```

Comment:
That's not how stubs are done, they need to be static inline, see other headers.
```
#if ...
void foo(void);
#else
static inline void foo(void)
{}
#endif
```

Reply:
done